test/test_bson.py

```diff
@@ -1053,6 +1054,57 @@ def test_unicode_decode_error_handler(self):
         self.assertRaises(InvalidBSON, decode, invalid_both, CodecOptions(
             unicode_decode_error_handler="junk"))
 
+    def round_trip_pickle(self, object, pickled_with_older=None):
```

Comment:
`pickled_with_older` should never be None. Please add it for all the types.

Reply:
Done.